packages/search/src/users/queries.ts

```diff
@@ -0,0 +1,77 @@
+/* Copyright (c) 2021 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
+import { gql } from "graphql-request";
```

Comment:
Is this the proper way to definie and export graphql queries or should this be done in a `.graphql` file?